doc/application/index.rst

```diff
@@ -1522,6 +1522,173 @@ After running the preprocessor, the final device tree used in the
 build is created by running the device tree compiler, ``dtc``, on the
 preprocessor output.
 
+Building and Configuring multiple images
+****************************************
+
+When firmware update and bootloaders are involved in an embedded
+application the final firmware that is flashed onto the device will
+usually need to consist of not one executable, but multiple
+executables that chain-load, or boot, each other. This section
+describes how to build and configure such a Zephyr application.
+
+When and why to use multiple images
+===================================
+
+First off, it's necessary to have a basic understanding of what
+exactly an executable is and why one would want to have more than one
+of them. An executable goes by other names depending on context, like
+program, image, or elf file. From now on we will be using the name
+image. From these names, you probably already know what an image is,
+but in case not; An image consists of pieces of code and data that
+goes by image-unique names recorded in a single symbol table. The
+symbol table exists as metadata in an .elf or .exe file and is not
+included when converted to a hex file for device flashing. Object
+files also consist of symbols, code, and data, but object files are
+not images, because unlike object files, images have had their code
+and data placed at addresses by a linker. In the end it is the linker
+that creates images. If confused, and you want to determine if you
+have zero, one, or more images, count the number of times the linker
+has been run.
+
+From this definition it follows that using two images can give you two
+opportunities. Firstly, it allows you to run the linker multiple
+times, and in this process parition the final firmware into several
+regions, as is often desired by bootloaders. And secondly, since there
+will be multiple symbol tables, it allows the same symbol name to
+exist multiple times in the final firmware. This again is useful for
+bootloader images, as they often need to have their own fixed copies
+of the same libraries that the application image is using.
+
+Enabling and configuring multiple images
+========================================
+
+Images are organized in a tree of parent-child relationships. Meaning,
+a parent image needs and enables through Kconfig one or more child
+images. Usually, this tree is branchless, a linked list of images
+where each image enables the child image that boots it. The simplest
```

Comment:
parent boots the child image, right?  If so, change ``that boots it.`` to ``and boots it.``
```